examples/d3/ivp_shell_convection/shell_convection.py

```diff
@@ -0,0 +1,125 @@
+"""
```

Comment:
Please add a line here noting that "This script is directly copied from the Dedalus 3 repository, see https://github.com/DedalusProject/dedalus/tree/master/examples/ivp_shell_convection"